Intersect.Server.Core/Entities/Events/CommandProcessing.cs

```diff
@@ -323,6 +323,17 @@ Stack<CommandInstance> callStack
                     }
                 }
             }
+
+            if (command.AllInInstance)
```

Comment:
Doesn't this need an `else` (which I think is the code right above this?)

Otherwise this runs n+1 times instead of n times.

Reply:
redid this, will just enqueue the common event for other players i.e a triggered common event normally would.